databuilder/models/user.py

```diff
@@ -37,7 +37,7 @@ def __init__(self,
                  email: str,
                  first_name: str = '',
                  last_name: str = '',
-                 name: str = '',
+                 full_name: str = '',
```

Comment:
Please Note: One of the sample files refers this as full_name and I thought that name made more sense. Up to you all if it should be name.